tsconfig.json

```diff
@@ -11,7 +11,9 @@
     "resolveJsonModule": true,
     "strict": true,
     "sourceMap": true,
-    "target": "es2019"
+    "target": "es2019",
+    // remove "skipLibCheck": true once issues w/ @perp/sdk-curie are resolved.
+    "skipLibCheck": true
```

Comment:
Does this silence the build errors with the Perp sdk?

Reply:
Yes, https://www.typescriptlang.org/tsconfig/skipLibCheck.html